lib/tldr-lint.js

```diff
@@ -180,3 +183,19 @@ linter.process = function(page, verbose, alsoFormat) {
 
   return result;
 };
+
+linter.processFile = function(file, verbose, alsoFormat) {
+  var result = linter.process(fs.readFileSync(file, 'utf8'), verbose, alsoFormat);
+  var filename = path.basename(file);
+  if (filename.substring(filename.length - 3) !== '.md') {
+    result.errors.push({ locinfo: { first_line: '0' }, code: 'TLDR107', description: this.ERRORS.TLDR107 });
+  } else {
+    // trim the extension off for checking against TLDR108
+    filename = filename.substring(0, filename.length - 3);
```

Comment:
And I guess the same here with using `path.basename(file, '.md')`.

Reply:
Good point, done.